helm-chart/renku-core/values.yaml

```diff
@@ -31,6 +31,13 @@ service:
 ingress:
   enabled: false
 
+metrics:
+  enabled: true
```

Comment:
usually the convention is to have metrics exporting default to false - can we do the same here?